Src/Plugins/Security/SEC101_045.PostgreSqlConnectionStringValidator.cs

```diff
@@ -18,6 +19,14 @@ public class PostgreSqlConnectionStringValidator : ValidatorBase
         internal static IRegex RegexEngine;
         private const string PortRegex = @"(?i)Port\s*=\s*(?<port>[0-9]{1,5})";
         private const string DatabaseRegex = @"(?i)(database|db)\s*=\s*(?<database>[^;]+)";
+        private const string PortKey = "PORTKEY";
+        private const string DatabaseKey = "DATABASEKEY";
```

Comment:
are we using this?

Reply:
Removed.

---
In reply to: [575575844](https://github.com/microsoft/sarif-pattern-matcher/pull/189#discussion_r575575844) [](ancestors = 575575844)